package/firmware/ath10k-firmware/Makefile

```diff
@@ -178,6 +194,13 @@ $(Package/ath10k-firmware-default)
   CATEGORY:=Firmware
 endef
 
+define Package/ath10k-firmware-qca9888-ct
+$(Package/ath10k-firmware-default)
+  TITLE:=ath10k CT 10.4.3 firmware for QCA9988 devices
```

Comment:
Sorry for that. Now I fix it, https://github.com/lede-project/source/pull/1088/commits/78d9bc7259673eb1e8f07a77fe25dd448259ac8f Thanks